### PR TITLE
changing the cron entries, because they were lacking the names...

### PR DESCRIPTION
**Pull request by ivanhorvath**
… and adding defattr to files section in one of the packages in the script spec file

### CODE REVIEW

**Comment by kwoodson** [comment]
:+1:

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 1]
```diff
@@ -198,15 +198,15 @@ host_monitoring_cron:
...
 
 - name: send etcd connections count on port 2380
   minute: "*/5"
-  job: ops-runner -f -s 30 -n /usr/bin/cron-send-connection-count.py etcd 2380 openshift.master.etcd.port_2380.connections.established
+  job: ops-runner -f -s 30 -n cscc.openshift.master.etcd.connections.2380 /usr/bin/cron-send-connection-count etcd 2380 openshift.master.etcd.port_2380.connections.established
 
 - name: send master api server connections count on port 443
   minute: "*/5"
-  job: ops-runner -f -s 30 -n /usr/bin/cron-send-connection-count.py openshift 443 openshift.master.api.port_443.connections.established
+  job: ops-runner -f -s 30 -n cscc.openshift.master.api.connections.ssl /usr/bin/cron-send-connection-count openshift 443 openshift.master.api.port_443.connections.established
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: I'd rather have this be `cscc.openshift.master.api.connections.443` to match the pattern.

ssl is generic. We can use ssl on any port. https would be better, but I still prefer the explicit 443.

**Comment by twiest** [comment]
:+1:

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 (hunk 1 above)
Comment (ivanhorvath) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Changed it, pull request is updated, want to give it a thumbs up Thomas?

**Pull request merged**